docker-compose/dse-6.8/docker-compose.yml

```diff
@@ -47,30 +49,6 @@ services:
       timeout: 10s
       retries: 10
 
-  dse-3:
```

Comment:
To compensate for additional memory on DSE nodes, reducing cluster size from 3 to 2.

Reply:
Totally agree with this